perfeed/tools/pr_summarizer.py

```diff
@@ -41,23 +39,21 @@ async def run(self, repo: str, pr_number: int) -> Tuple[PRSummary, PRSummaryMeta
 
         summary = self.llm.chat_completion(system_prompt, user_prompt)
         curated_summary = json_output_curator(summary)
+
+        print(f"Summary of PR {pr.number}: \n{curated_summary}\n")
+
         pr_summary = PRSummary(**json.loads(curated_summary))
-        current_time = datetime.now(timezone.utc)
-        pr_metadata = PRSummaryMetadata(
```

Comment:
This will be added later in `WeeklySummarizer` to maintain the mapping

Reply:
how do u mean? PRSummaryMetadata is needed to save the each PRsummary

Reply:
Yes, and I'll construct each `PRSummaryMetadata` and associate it with the corresponding `PRSummary` in `WeeklySummarizer`